learning_resources/etl/utils.py

```diff
@@ -330,16 +330,17 @@ def documents_from_olx(
                     filebytes = f.read()
 
                 mimetype = mimetypes.types_map.get(extension_lower)
+                checksum = md5(filebytes).hexdigest()  # noqa: S324
 
                 yield (
                     filebytes,
                     {
-                        "key": f"{path}/{filename}",
+                        "key": checksum,
```

Comment:
curious if there are unexpected cases here since it seems like the "key" is unique with the associated run. are there cases where we could have 2 separate files in the same run that have the same content?

Reply:
The scenario you are describing is exactly what this pr is fixing - we found that there are duplicate files with different file names and the same content in lots of edx course folders. This pr fixes that issue.

There is an edge case where files with different content that isn't parsed fully by tikka for whatever reason might end up looking like the same (almost blank) file. But since those files don't have useful content anyway, that's ok